dotcom-rendering/src/web/components/TextBlockComponent.tsx

```diff
@@ -209,69 +210,129 @@ const styles = (format: ArticleFormat) => css`
 	}
 `;
 
+const buildElementTree =
+	(
+		html: string,
+		format: ArticleFormat,
+		isFirstParagraph: boolean,
+		forceDropCap?: boolean,
+	) =>
+	(node: Node, key: number): ReactNode => {
+		const showDropCaps = shouldShowDropCaps(
+			html,
+			format,
+			isFirstParagraph,
+			forceDropCap,
+		);
```

Comment:
I think `showDropCaps` could be passed as a parameter to `buildElementTree` as I think `showDropCaps` works on the raw HTML?